frontend/packages/kubevirt-plugin/src/components/vm-disks/disk-summary.tsx

```diff
@@ -9,45 +9,30 @@ import { WINTOOLS_CONTAINER_NAMES } from '../modals/cdrom-vm-modal/constants';
 import { VMKind } from '../../types';
 import { V1Disk } from '../../types/vm/disk/V1Disk';
 
-const DiskSummaryRow: React.FC<DiskSummaryRowProps> = ({ title, value }) => (
-  <div className="kubevirt-disk-summary__disk">
-    <div id={`kubevirt-disk-summary-disk-title-${title}`}>{title}</div>
-    <div id={`kubevirt-disk-summary-disk-value-${value}`}>{value}</div>
-  </div>
-);
-
 export const DiskSummary: React.FC<DiskSummaryProps> = ({ disks, vm }) => (
   <div className="kubevirt-disk-summary">
-    {disks.map(({ name }, i) => {
+    {disks.map(({ name }) => {
       const container = getContainerImageByDisk(vm, name);
       const pvc = getPVCSourceByDisk(vm, name);
       const url = getURLSourceByDisk(vm, name);
+      let value = '';
 
       if (_.includes(WINTOOLS_CONTAINER_NAMES, container)) {
-        return (
-          <DiskSummaryRow
-            key={`disk-summary-${name}`}
-            title={`Drive ${i + 1}: Windows Tools`}
-            value={container}
-          />
-        );
-      }
-      if (container) {
-        return (
-          <DiskSummaryRow
-            key={`disk-summary-${name}`}
-            title={`Drive ${i + 1}: Container`}
-            value={container}
-          />
-        );
-      }
-      if (url) {
-        return (
-          <DiskSummaryRow key={`disk-summary-${name}`} title={`Drive ${i + 1}: URL`} value={url} />
-        );
+        value = `Windows Tools: ${container}`;
+      } else if (container) {
+        value = `Container: ${container}`;
+      } else if (url) {
+        value = `URL: ${url}`;
+      } else if (pvc) {
+        value = `PVC: ${pvc}`;
       }
+
       return (
-        <DiskSummaryRow key={`disk-summary-${name}`} title={`Drive ${i + 1}: PVC`} value={pvc} />
+        <div className="kubevirt-disk-summary__disk">
```

Comment:
- is this classname up to date?
- maybe we can add an id for the whole group from the name
- missing a key

Reply:
- removed class, It looks unused.
  - added an id
  - added a key